test/test_gpu_outer_reduction.cpp

```diff
@@ -32,10 +32,12 @@
 
 namespace nvfuser {
 
+using OuterReductionTest = NVFuserTest;
+
 using namespace at::indexing;
 
 // Shmoo testing of the optimized grouped grid welford
-TEST_F(NVFuserTest, FusionGroupedGridWelfordOuterOpt_CUDA) {
+TEST_F(OuterReductionTest, FusionGroupedGridWelfordOuterOpt_CUDA) {
```

Comment:
Could you also shorten the name of each test like:

```suggestion
TEST_F(OuterReductionTest, GroupedGridWelfordOuterOpt) {
```

The `Fusion` prefix and `_CUDA` suffix were old conventions that are no longer necessary